### PR TITLE
Remove timm as dependency

### PR DESCRIPTION
**Pull request by alvanli**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -35,8 +35,8 @@ requests==2.26.0
 six==1.16.0
 text-unidecode==1.3
 toml==0.10.2
-torch==1.9.0
-torchvision==0.10.0
+torch==1.11.0
```
Comment (NeelayS) [inline requirements.txt:38]: We recently made the requirements for torch and related packages more lax by only requiring that the package version be greater than or equal to some version. I think we should keep that.

**Review by NeelayS — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (alvanli) [reply requirements.txt]: [4a91923](https://github.com/SforAiDl/vformer/pull/88/commits/4a9192385762f003d1644d3d06d9836eaa2531df)

**Review by NeelayS — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (NeelayS) [reply requirements.txt]: @alvanli We can keep the versions for torch as 1.9.0 and torchvision as 0.10.0, right? No need to bump them.

Comment (NeelayS) [reply requirements.txt]: @alvanli Just to confirm, is `StochasticDepth` only available from version 0.11.0 onwards in torchvision?

**Comment by abhi-glitchhg** [comment]
Can we give user some more flexibility by having `mode` as an input parameter of the class?

Other than that it looks good to merge.

**Review by abhi-glitchhg — APPROVED** [review]
Thanks @alvanli

**Comment by NeelayS** [comment]
> Can we give user some more flexibility by having `mode` as an input parameter of the class?

Yeah, I guess we can have a `drop_path_mode` parameter. What do you think?

**Comment by NeelayS** [comment]
Looks good, merging!

**Pull request merged**